cmd.sh

```diff
@@ -3,6 +3,6 @@
 if [ "$APP_ENV" == "DEVELOPMENT" ] || [ -z "$APP_ENV" ]; then
     gunicorn -w 4 -b 0.0.0.0:8081 wsgi:app --reload --worker-class gevent --timeout 600  # nosec
 else
-  pipenv run python wsgi.py
-  # gunicorn -b 0.0.0.0:8081 -w 4 wsgi:app --worker-class gevent
+  sleep 10
```

Comment:
Can this be removed?

Reply:
We need so their is no race condition in docker-compose

Reply:
Race condition for the database connection?

Reply:
Yes

Reply:
So we have had a few patterns for waiting for the database connection --

This one uses python to wait - https://github.com/brighthive/data-resource-api/blob/084b5e39ca0b874bf2a00a1158ebe1b0001b7dcf/data_resource_api/app/data_managers/data_resource_manager.py#L100-L144

This one uses bash (may be a poor fit?) - https://github.com/vishnubob/wait-for-it

This one uses bash to call a python function:

https://github.com/brighthive/jdx-api/blob/ee6ee2d88e808fadbfbb848e5c49c4f9253a667d/run.sh#L11-L19

https://github.com/brighthive/jdx-api/blob/master/jdxapi/wait_for_db.py

I can add this as a bug and whoever wants to tackle it can take care of it.

Reply:
https://app.clubhouse.io/brighthive/story/1260/drg-should-wait-for-database-in-docker-mode